src/shared/lib/oql/oql-parser.pegjs

```diff
@@ -1,35 +1,58 @@
 start
-	= Query
-	/ sp { return false; }
-
-NaturalNumber = number:[0-9]+ { return number.join("");}
-Number = "-" number: Number { return "-"+number;}
-        / whole_part:NaturalNumber "." decimal_part:NaturalNumber { return whole_part + "." + decimal_part;}
-        / "." decimal_part:NaturalNumber { return "."+decimal_part;}
-        / whole_part:NaturalNumber {return whole_part;}
-String = word:[-_.@/a-zA-Z0-9*]+ { return word.join("") }
+  = Query
+  / sp { return false; }
+
+NaturalNumber = number:[0-9]+ { return number.join(""); }
+
+Number
+  = "-" number:Number { return "-" + number; }
+  / whole_part:NaturalNumber "." decimal_part:NaturalNumber {
+      return whole_part + "." + decimal_part;
+    }
+  / "." decimal_part:NaturalNumber { return "." + decimal_part; }
+  / whole_part:NaturalNumber { return whole_part; }
+
+String = word:[-_.@/a-zA-Z0-9*]+ { return word.join(""); }
+
 MutationProteinChangeCodeChar = char:[-_./a-zA-Z0-9*] { return char; }
+
 AminoAcid = letter:[GPAVLIMCFYWHKRQNEDST] { return letter; }
+
 // any character, except " :
-StringExceptQuotes = stringExceptQuotes:[^"]+ { return stringExceptQuotes.join("") }
+StringExceptQuotes
+  = stringExceptQuotes:[^"]+ { return stringExceptQuotes.join(""); }
 
 sp = space:[ \t\r]+
+
 msp = space:[ \t\r]*
 
 zmbs = zero_or_more_breaks_and_spaces:[,; \t\r\n]*
+
 ombs = one_or_more_breaks_and_spaces:[,; \t\r\n]+
 
 StartMergedGenes
-	= "[" zmbs "\"" label:StringExceptQuotes "\"" {return {"label": label, "list":[]};}
-	/ "[" zmbs {return {"label": undefined, "list":[]};}
+  = "[" zmbs "\"" label:StringExceptQuotes "\"" {
+      return { label: label, list: [] };
+    }
+  / "[" zmbs { return { label: undefined, list: [] }; }
 
 // Case-insensitive keywords
 AMP = "AMP"i
-HOMDEL = "HOMDEL"i
+
+HOMDEL
+  = "HOMDEL"i
+  / "DeepDel"i
+
 GAIN = "GAIN"i
-HETLOSS = "HETLOSS"i
+
+HETLOSS
+  = "HETLOSS"i
+  / "ShallowDel"i
+
```

Comment:
These were the most significant changes in the OQL grammar, the rest of the changes are from an open-source prettier formatter for pegjs.